spotbugs/etc/messages_ja.xml

```diff
@@ -99,72 +99,135 @@
   <BugCategory category="CORRECTNESS">
     <Description>正確性</Description>
     <Abbreviation>C</Abbreviation>
-    <Details>バグの可能性 - おそらく、開発者が意図していなかったコードになっている明らかなコーディング間違いです。
-      我々は低い誤検出率のために努力します。
+    <Details>
+<![CDATA[
+<p>
+バグの可能性 - おそらく，開発者が意図していなかったコードになっている明らかなコーディングミスです。

```

Comment:
Could you tell me your intent to replace `、` with `， `?

I thought that you chose report type style, but you still use `。`